cmd/print.go

```diff
@@ -59,12 +74,46 @@ func getResourcePrinter(outputFormat string) (printers.ResourcePrinter, error) {
 	}
 }
 
+// getNamespaceFilter returns a namespace filter, taking into consideration whether a specific
+// namespace is requested, or all of them are.
+func getNamespaceFilter(requestedNamespace string, useAllNamespaces bool) (string, error) {
```

Comment:
this func should be changed if you choose to add MarkFlagsMutuallyExclusive (as per my below comment)

Reply:
I've added MarkFlagsMutuallyExclusive, but I don't see how it should change the functionality at getNamespaceFilter...
We'll still need to figure out what is the required namespace filter based on those two flags, regardless of whether or not they both get assigned simultaneously.

Let me know if you got another idea in your mind.

Reply:
right, added just small nits to the comments